core/chains/evm/logpoller/orm.go

```diff
@@ -716,23 +761,26 @@ func (o *DbORM) SelectIndexedLogsByBlockRange(start, end int64, address common.A
 	if err != nil {
 		return nil, err
 	}
-	var logs []Log
-	err = o.q.WithOpts(qopts...).SelectNamed(&logs, `
-		SELECT * FROM evm.logs 
+
+	query := `SELECT * FROM evm.logs 
 				WHERE evm_chain_id = :evm_chain_id 
 				AND address = :address
 				AND event_sig = :event_sig
 				AND topics[:topic_index] = ANY(:topic_values)
 				AND block_number >= :start_block
 				AND block_number <= :end_block
-				ORDER BY (block_number, log_index)`, args)
+				ORDER BY (block_number, log_index)`
+
+	var logs []Log
+	query, sqlArgs, _ := o.db.BindNamed(query, args)
```

Comment:
There are many more unchecked errors in this file

Reply:
Fixed